src/main/java/duke/tasklist/DukeList.java

```diff
@@ -0,0 +1,131 @@
+package duke.tasklist;
+
+import duke.storage.Storage;
+import duke.task.Deadlines;
+import duke.task.Events;
+import duke.task.Task;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public class DukeList {
+
+    private List<Task> a;
```

Comment:
Perhaps rename this variable to be more descriptive about it's usage (e.g. "tasks)?